packages/consumption/test/modules/requests/itemProcessors/proposeAttribute/ProposeAttributeRequestItemProcessor.test.ts

```diff
@@ -342,31 +355,31 @@ describe("ProposeAttributeRequestItemProcessor", function () {
             });
         });
 
-        test("returns an error when the given Attribute id belongs to a peer Attribute", async function () {
-            const someOtherIdentity = CoreAddress.from("id1");
+        test("returns an error when the existing IdentityAttribute is already shared", async function () {
```

Comment:
This test moved as "returns an error when the given Attribute id belongs to a peer Attribute" to the file validateAttributeMatchesWithQuery.test.ts in the describe block "describe("ProposeAttributeRequestItem with IdentityAttributeQuery")", because this error is now thrown by the auxiliary function validateAttributeMatchesWithQuery.